pkg/imagevector/imagevector-packr.go

```diff
@@ -0,0 +1,8 @@
+// +build !skippackr
+// Code generated by github.com/gobuffalo/packr/v2. DO NOT EDIT.
+
+// You can use the "packr clean" command to clean up this,
+// and any other packr generated files.
+package imagevector
+
+import _ "github.com/gardener/gardener-extension-networking-cilium/pkg/imagevector/packrd"
```

Comment:
Similar to https://github.com/gardener/gardener-extension-shoot-cert-service/pull/97#discussion_r740774455?

Why packr is added again?

Reply:
Thanks for noticing, not sure what happened here. I guess I had an old version with `packr` and even though I followed what I think is the correct way to revendor (`make revendor`, `make generate`, etc.), it somehow not only remained, but managed to pass all tests executed by `make verify`. Since I opened many revendor PRs at the same time I followed the same procedure for all of them but looked carefully at the changes for only some of them. Fixed now.